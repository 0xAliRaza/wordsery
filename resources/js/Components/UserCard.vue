<template>
    <div>
        <!-- By <a href="#">{{ user.name }}</a> at <span>{{ created_at }}</span> -->
    </div>
</template>

<script>
export default {
    // props: {
    //     user: {type: Object, required: true},
    //     created_at: {type: String, required: true},
    // },
    setup() {

    },
}
</script>

<style lang="scss" scoped></style>
