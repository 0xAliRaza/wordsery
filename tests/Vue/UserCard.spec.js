import { shallowMount } from "@vue/test-utils";
import UserCard from "@/Components/UserCard";

const defaultProps = {
    user: { name: "TEST_USER" },
    created_at: "TEST_CREATED_AT",
};
const factory = (props = defaultProps) => {
    return shallowMount(UserCard, {
        props: { ...props },
    });
};

describe("UserCard.vue", () => {
    it("renders correctly", () => {
        const wrapper = factory();
        expect(wrapper.html()).toMatchSnapshot();
    });
});
