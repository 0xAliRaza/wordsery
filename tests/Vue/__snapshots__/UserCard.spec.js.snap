// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCard.vue renders correctly 1`] = `<div> By <a href="#">TEST_USER</a> at <span>TEST_CREATED_AT</span></div>`;
